src/index.js

```diff
@@ -220,9 +222,9 @@ class Gnosis {
              */
             (async () => {
                 if(await utils.promisify(this.web3.version.getNetwork)() == 1) {
-                    this.etherToken = this.contracts.EtherToken.at('0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2')
+                    this.etherToken = this.contracts.WETH9.at(weth9MainnetAddress)
```

Comment:
If using canonical-weth package, can skip this check as the correct address is written in the artifact.

Reply:
Nice, pushed the changes